README-RU.md

```diff
@@ -0,0 +1,113 @@
+[![Pub](https://img.shields.io/pub/v/flame.svg?style=popout)](https://pub.dartlang.org/packages/flame) ![Test](https://github.com/flame-engine/flame/workflows/Test/badge.svg?branch=master&event=push) [![Discord](https://img.shields.io/discord/509714518008528896.svg)](https://discord.gg/pxrBmy4)
```

Comment:
we should extract this to a separated PR @lig

Reply:
Oh boy. I didn't notice. Agree. @unger1984 can you do this?

Reply:
It`s was merged in PR #551

Reply:
> It`s was merged in PR #551

I see. It could be that I've rebased my branch on `develop`.

Reply:
@luanpotter could you please move the `null-safety` branch forward with `develop`?

Reply:
@lig updated it now, there will most likely be several branches merged to develop today though.
If you ping me tomorrow I can update it again if I forget.

Reply:
@spydon wilco, thanks! After this PR merged `null-safety` will be diverted and each rebase is going to be a separate PR, I guess.